hunts/views.py

```diff
@@ -162,8 +162,8 @@ def post(self, request, pk):
             if google_api_client:
                 sheet = google_api_client.create_google_sheets(name)
             else:
-                # TODO(erwa): This should incur a warning.
-                sheet = puzzle_url
+                logger.warn("Sheet not created for puzzle %s" % name)
+                sheet = None
```

Comment:
Do we need to update any templates to account for this? I recall that the site defaulted to the puzzle URL when a sheet wasn't available, so I'm wondering if this might break something here: https://github.com/cardinalitypuzzles/smallboard/blob/cba90c7c6fc8490727895695b9083ec26fa87a2e/hunts/templates/all_puzzles.html#L347

Reply:
I tested this-- I think the front-end already handles this correctly: 
![image](https://user-images.githubusercontent.com/3475509/100275710-38481980-2f15-11eb-8d98-992ad1437a32.png)

Reply:
Ah perfect - thanks for checking!